src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxTreeTests.cs

```diff
@@ -21,21 +22,29 @@ public class SyntaxTreeTests : ParsingTests
     {
         public SyntaxTreeTests(ITestOutputHelper output) : base(output) { }
 
-        // Diagnostic options on syntax trees are now obsolete
```

Comment:
I would prefer to leave this `pragma`, rather than mark tests as obsolete.

Reply:
The attribute on a test declares that the test is testing obsolete API. Seems better to have that declared explicitly then just suppress warning for a set of tests.